website/docs/r/dl_gateway_virtual_connection.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: "ibm"
+page_title: "IBM : dl_gateway_virtual_connection"
+sidebar_current: "docs-ibm-resource-dl-gateway-virtual-connection"
+description: |-
+  Manages IBM Direct Link Gateway Virtual Connection.
+---
+
+# ibm\_dl_gateway_vc
+
+Provides a direct link gateway virtual connection resource. This allows direct link gateway virtual connection to be created, and updated and deleted.
+
+## Example Usage
+
+```hcl
+resource "ibm_dl_gateway_vc" "test_dl_gateway_vc"{
```

Comment:
Format the the example code..copy the example cofig to some temp.tf and run terraform fmt temp.tf